descriptors/ModuleDescriptor-template.json

```diff
@@ -440,7 +440,7 @@
     },
     {
       "id": "patron-action-session",
-      "version": "0.1",
+      "version": "1.1",
```

Comment:
This API is marked experimental, why move it to a non-experimental API for this change?

Reply:
updated version